src/cljs/rems/language.cljs

```diff
@@ -2,11 +2,22 @@
   (:require [clojure.string :as str]
             [re-frame.core :as rf :refer [reg-event-fx reg-sub]]))
 
+(def ^:private language-cookie-name "rems-user-preferred-language")
+
+(defn get-language-cookie []
+  (when-let [value (.. js/document -cookie)]
+    (keyword (second (str/split value #"=")))))
+
+(defn- set-language-cookie! [language]
+  (let [year-from-now (.setFullYear (js/Date.) (inc (.getFullYear (js/Date.))))]
```

Comment:
cljs-time?

Reply:
It would be many more characters because of the requires and formatting. This is as simple as it gets?

Reply:
I would've preferred the `(plus (now) (years 1))` of cljs-time, but this is fine too.

Reply:
Btw. I fixed a problem from here but it still stands that cljs-time is not simpler in this.